pkg/controller/data_download_controller.go

```diff
@@ -416,6 +425,43 @@ func (r *DataDownloadReconciler) findSnapshotRestoreForPod(podObj client.Object)
 	return requests
 }
 
+func (r *DataDownloadReconciler) FindDataDownloads(ctx context.Context, cli client.Client, ns string) ([]*velerov2alpha1api.DataDownload, error) {
+	pods := &v1.PodList{}
+	var dataDownloads []*velerov2alpha1api.DataDownload
+	if err := cli.List(ctx, pods, &client.ListOptions{LabelSelector: labels.Everything(), Namespace: ns}); err != nil {
```

Comment:
Use `velerov1api.DataDownloadLabel` as the `LabelSelector` instead of `labels.Everything()`. Otherwise, it will return all pods without any filtering.